modules/core/src/shadertools/src/lib/platform-defines.js

```diff
@@ -80,7 +80,9 @@ export function getVersionDefines(gl, glslVersion, isFragment) {
 #endif
 `;
   }
-  if (hasFeatures(gl, FEATURES.GLSL_FRAG_DATA)) {
+  if (hasFeatures(gl, FEATURES.GLSL_FRAG_DATA) &&
+  canCompileGLGSExtension(gl, FEATURES.GLSL_FRAG_DATA, {behavior: 'require'})
```

Comment:
This runs for all platforms right? Does it run only once?

Reply:
`canCompileGLGSExtension` runs only once per extension as it caches the result. And it always return true for all non IE browsers, so this change affects only IE browsers.